extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -302,7 +302,7 @@ export async function runRemoteQuery(
       message: 'Sending request'
     });
 
-    const workflowRunId = await runRemoteQueriesApiRequest(credentials, 'main', language, repositories, owner, repo, base64Pack, dryRun);
+    const workflowRunId = await runRemoteQueriesApiRequest(credentials, 'better-errors', language, repositories, owner, repo, base64Pack, dryRun);
```

Comment:
Just checking we're intentionally checking this in. Fine if we decided to do that!

Reply:
This PR is broken unless we point to the `better-errors` branch. When that branch is merged on qc-run2, then we can change this back to `main`. I'll add a comment to that effect.